spec/models/company_value_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe CompanyValue, type: :model do
+  it 'has a valid factory' do
+    expect(build(:company_value)).to be_valid
+  end
+
+  context 'when title is nil' do
+    it { is_expected.to validate_presence_of(:title) }
```

Comment:
in this context title is not `nil` - if no subject is specified it is creating default values from the factory